tools/initRepo/lib/initRepo.ts

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Import most things with types if possible.
+import Opts = require('node-getopt');
+import * as _ from 'lodash';
+import * as Promise from 'bluebird';
+
+const hmac = require('crypto');
```

Comment:
There's linting checks that will catch this in ts-lint I think. Might be time to come up with a standard tsconfig and linting setup soon...